nui-gestalt-di/build.gradle

```
@@ -0,0 +1,112 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.github.classgraph:classgraph:4.8.104'
+    }
+}
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+ext {
+    // classes, interfaces, annotations, which should be adds to index for gestalt's ClassIndex
+    indexingClasses = [
+            "org.terasology.nui.UIWidget",
+            "org.terasology.nui.UILayout",
+            "org.terasology.reflection.copy.CopyStrategy"
+    ]
+}
+
+apply from: "$rootDir/gradle/common.gradle"
+
+repositories {
+    google()
+}
+
+dependencies {
+    api group: 'org.terasology.gestalt', name: 'gestalt-module', version: '7.0.7-SNAPSHOT'
+    api group: 'org.terasology.gestalt', name: 'gestalt-asset-core', version: '7.0.7-SNAPSHOT'
+    api group: 'org.terasology.gestalt', name: 'gestalt-di', version: '7.0.7-SNAPSHOT'
```

`gestalt-di` is now a part of gestalt `8.0.0-SNAPSHOT`, so these should be changed.
```suggestion
    api group: 'org.terasology.gestalt', name: 'gestalt-module', version: '8.0.0-SNAPSHOT'
    api group: 'org.terasology.gestalt', name: 'gestalt-asset-core', version: '8.0.0-SNAPSHOT'
    api group: 'org.terasology.gestalt', name: 'gestalt-di', version: '8.0.0-SNAPSHOT'
```